docs/marketplace/overview_marketplace.md

```diff
@@ -18,6 +18,12 @@ The Marketplace components types you can use include:
 
 * **[Application](/marketplace/applications/mia_applications.md)**: a bundle of resources including Plugins, Templates, and Examples, but also Endpoints, Collections, and Public Variables.
 
+* **[Proxy](/development_suite/api-console/api-design/proxy.md)**: specific configurations used to invoke APIs that are external to the current project, but may be provided by an external service or another project.
+
+* **[Sidecars](/software-catalog/manage-items/mia-ctl/create/create-item-by-type/create_sidecar.md)**: secondary utility containers running side by side with the main container in the same host.
+
+* **[Infrastructure Resources](/software-catalog/manage-items/mia-ctl/create/create-item-by-type/create_infrastructure_resource.md)**: custom objects that are not part of the standard Console supported resources.
+
```

Comment:
Technically we also have Console extensions in the marketplace. Do we want to leave them out to not cross over the Extension Manager?